PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1665,7 +1666,7 @@ Find in all files but exclude all folders log or logs recursively:
 *.* !+\log*"/><!-- HowToReproduce: Tip of mouse hovered on "Filters" label in "Find in Files" section of Find dialog. -->
 			<find-in-files-select-folder value="Select a folder to search from"/><!-- HowToReproduce: Search > Find in Files > [...] -->
 			<find-status-top-reached value="Find: Found the 1st occurrence from the bottom. The beginning of the document has been reached."/>
-			<find-status-end-reached value="Find: Found the 1st occurrence from the top. The end of the document has been reached."/>
+			<find-status-end-reached value="Find: Find: Found the last occurrence from the top. The end of the document has been reached."/>
```

Comment:
"Find:" is doubled.

Additionally, english.xml and englisch_customizable.xml should be synced. The customizable states "last occurrence" for top-reached and end-reached (find and replace) which seems correct to me.